### PR TITLE
chore: Add hooli partner demo page

### PR DESCRIPTION
**Pull request by iagormoraes**
[Ticket](https://trello.com/c/IRaclA1c/5452-build-new-demo-showing-partner-issuance-from-hooli-to-kredita)

## Summary
Adds hooli partner demo page and overview information that points to the related page.

Screenshot 2023-06-27 at 11 08 20
Screenshot 2023-06-27 at 11 08 26

## Changes
- add new page
- add new sidebar demo option
- add overview information

## Testing
Tested locally the pages.

## Checklist

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas _to a borderline excessive amount_
- [x] I have made any relevant corresponding changes to the documentation including the project readme.
- [x] I have run and tested the changes locally
- [ ] If it is a core feature, I have added an appropriate amount of unit tests.
- [ ] Any dependent changes have been merged and published in upstream projects

### CODE REVIEW

File: docs/hooli-partner-demo.mdx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+id: hooli-partner-demo
+title: Hooli Partner Demo
+sidebar_label: Hooli Partner Demo
+slug: /hooli-partner-demo
+---
+
+import Tip from '../src/components/Tip';
+
+The open source Hooli partner demo demonstrates an example implementation of the issuing partner Verified Inc. digital identity cards which then redirects to Kredita. It is an application for a fictional customer, Hooli, which uses Verified Inc. to issue reusable <Tip type="credential">credentials</Tip> to its users and send them to the partner url.
```
Comment (jacobleesinger) [inline docs/hooli-partner-demo.mdx:10]:
```suggestion
The open source Hooli partner demo demonstrates an example implementation of issuing Verified Inc. digital identity cards to share with a specific partner. It is an application for a fictional customer, Hooli, which uses Verified Inc. to issue reusable <Tip type="credential">credentials</Tip> to its users and share them with another fictional customer, Kredita.
```

File: docs/hooli-partner-demo.mdx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Additional information on how to issue credentials with a partner can be found in the Guide's issuance with partner [documentation](/issuance-guide#partner-issuance).
```
Comment (jacobleesinger) [inline docs/hooli-partner-demo.mdx:12]:
```suggestion
Additional information on how to issue credentials with a partner can be found in the Partner Issuance [guide](/issuance-guide#partner-issuance).```

**Review by rsmets — APPROVED** [review]
Looks great to me. Thanks for doing this @iagormoraes. I am going to approve and merge but let's hold off an releasing until we have the Kredita side of the partner demo in place.

File: docs/hooli-partner-demo.mdx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## Web Demo
+The live Hooli web application demo can be found [here](https://hooli-partner-web.demo.sandbox-verifiedinc.com/login). Its open source code can be found on [Github](https://github.com/UnumID/Hooli-Demo-Web).
+
+Feel free to use a real email address that you control or our test account, richard@piedpiper.net. For the purposes of the demo, the password field is irrelevant. If opting to use the test account you can use the static OTP of `111111`. For the appropriate experience of demo, it is recommended to use the same e-mail from [Hooli](/holy-demo) and [Kredita](/kredita-demo).
```
Comment (jacobleesinger) [inline docs/hooli-partner-demo.mdx:17]: Note: I haven't used the new demo yet, so I'm not sure if we still need the part about using the same email or not.

```suggestion
Feel free to use a real email address that you control, or our test account, richard@piedpiper.net. You can enter any value in the password field. If you used a real email address, we will send you a one-time password to authenticate to the [wallet](https://wallet.sandbox-verifiedinc.com). If you used the test account, your OTP is `111111`.
```

**Review by jacobleesinger — COMMENTED** [review]
Some suggestions to improve readability.

**Comment by rsmets** [comment]
@iagormoraes I miss spoke... the kredita side of the demos does not need to be updated. However, still I think we should hold off on releasing until the partner hooli flavor of the demo is properly deployed to sandbox. 

Also, worth handling the Jacob's suggestions.

File: docs/hooli-partner-demo.mdx (hunk 2 above)
Comment (iagormoraes) [reply docs/hooli-partner-demo.mdx:12]: This was resolved in https://github.com/UnumID/UnumID.github.io/pull/102

File: docs/hooli-partner-demo.mdx (hunk 3 above)
Comment (iagormoraes) [reply docs/hooli-partner-demo.mdx:17]: This was resolved in https://github.com/UnumID/UnumID.github.io/pull/102

**Comment by iagormoraes** [comment]
> 

Let's hold then. 👍

**Pull request merged**